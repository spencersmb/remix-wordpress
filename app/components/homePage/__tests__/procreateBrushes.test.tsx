import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import ProcreateBrushes from '../procreateBrushes';

it('Should render Procreate Brushes Component ', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <ProcreateBrushes />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
})