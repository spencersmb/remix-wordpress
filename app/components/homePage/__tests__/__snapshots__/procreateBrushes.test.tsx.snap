// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Procreate Brushes Component  1`] = `
<section
  className="et-grid-basic bg-repeat overflow-hidden tablet:grid-row-[auto_auto] py-20"
  style={
    Object {
      "backgroundImage": "url(https://et-website.imgix.net/et-website/images/flower-pattern-01.jpg)",
    }
  }
>
  <div
    className="image-1 relative col-span-2 col-start-2 h-[350px] tablet:col-start-2 tablet:mt-8 tablet:col-span-8 tablet:row-start-2 tablet:h-[200px] laptop:mt-0 desktop:h-[410px] desktop:col-start-5 desktop:col-span-7 desktop:mr-16"
  >
    <div
      className="absolute top-0 left-0 tablet:left-auto tablet:right-0 w-[530px] desktop:w-[772px] desktopXl:w-[882px]"
    >
      <div
        className="relative max-w-[1000px] z-20"
      >
        <div
          className="absolute top-[-4.5%] left-[-5.6%] scale-[.82] w-full overflow-hidden rounded-md tablet:rounded-xl art z-20"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "74.92857142857143%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-align.jpg?auto=format&w=20&fit=clip&auto=compress)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "1049px",
                    "width": "1400px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "1049px",
                      "width": "1400px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
        <div
          className="relative z-10 ipad"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "73.3%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-device-1_3-min.png?auto=format?auto=format&w=20&fit=clip)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "733px",
                    "width": "1000px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "733px",
                      "width": "1000px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="image-2 hidden relative col-span-2 col-start-2 h-[350px] mt-8 tablet:col-start-10 tablet:col-span-4 tablet:row-start-2 tablet:h-[200px] tablet:block laptop:mt-0 desktop:col-start-11 desktop:col-span-4 desktop:h-[410px] desktopXl:ml-16"
  >
    <div
      className="absolute top-0 left-0 w-[530px] desktop:w-[772px] desktopXl:w-[882px]"
    >
      <div
        className="relative max-w-[1000px] z-20"
      >
        <div
          className="absolute top-[-4.5%] left-[-5.6%] scale-[.82] w-full overflow-hidden rounded-md tablet:rounded-xl art z-20"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "74.92857142857143%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-align.jpg?auto=format&w=20&fit=clip&auto=compress)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "1049px",
                    "width": "1400px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "1049px",
                      "width": "1400px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
        <div
          className="relative z-10 ipad"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "73.3%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-device-1_3-min.png?auto=format?auto=format&w=20&fit=clip)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "733px",
                    "width": "1000px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "733px",
                      "width": "1000px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="image-3 hidden relative col-span-2 col-start-2 h-[350px] tablet:col-start-9 tablet:col-span-5 tablet:row-start-1 tablet:h-[400px] tablet:block"
  >
    <div
      className="absolute top-[-40px] left-[-110px] w-[530px] rotate-90 scale-y-[-1] tablet:left-0 desktop:w-[772px] tablet:top-5 desktop:top-[-20px] desktop:left-[-120px] desktopXl:w-[882px] desktopXl:top-[-135px]"
    >
      <div
        className="relative max-w-[1000px] z-20"
      >
        <div
          className="absolute top-[-4.5%] left-[-5.6%] scale-[.82] w-full overflow-hidden rounded-md tablet:rounded-xl art z-20"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "74.92857142857143%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-align.jpg?auto=format&w=20&fit=clip&auto=compress)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "1049px",
                    "width": "1400px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "1049px",
                      "width": "1400px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
        <div
          className="relative z-10 ipad"
        >
          <div
            className="relative flex-1"
            data-testid="imigx-padding-bot"
            style={
              Object {
                "paddingBottom": "73.3%",
              }
            }
          >
            <div
              className="absolute w-full lazy-load-wrapper lazy-load-wrapper-block lazy-load-image-full transition-all duration-300"
              data-testid="imigx-container"
            >
              <span
                className=" lazy-load-image-background blur"
                style={
                  Object {
                    "backgroundImage": "url(https://et-website.imgix.net/et-website/images/ipad-device-1_3-min.png?auto=format?auto=format&w=20&fit=clip)",
                    "backgroundSize": "100% 100%",
                    "color": "transparent",
                    "display": "inline-block",
                    "height": "733px",
                    "width": "1000px",
                  }
                }
              >
                <span
                  className=""
                  style={
                    Object {
                      "display": "inline-block",
                      "height": "733px",
                      "width": "1000px",
                    }
                  }
                />
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="flex flex-col items-start col-span-2 col-start-2 px-8 py-8 mt-16 tablet:row-start-1 tablet:col-start-3 tablet:col-span-6 tablet:my-[78px] tablet:pb-8 laptop:mt-36 desktop:mb-32 desktop:px-16 desktop:pb-16 desktop:col-start-2 desktop:col-span-6 desktop:max-w-[725px] bg-white"
  >
    <h3
      className="relative mb-8 text-3xl font-sentinel__SemiBoldItal tablet:text-4xl laptop:text-[2.75rem] laptop:leading-[3rem] desktop:text-6xl"
    >
      <div
        className="left-[-22px] tablet:!top-[-59px] text-4xl tablet:left-[-52px] text-[#FF551F] absolute font-bonVivant text-2xl tablet:text-5xl italic top-[-47px] left-[-25px] rotate-[-14deg]"
      >
        Brand New!
      </div>
      Make inspiring art using custom procreate brushses!
    </h3>
    <a
      className="btn btn-primary btn-xl btn-primary-ring"
      href="/products"
      onBlur={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
      onTouchStart={[Function]}
    >
      View my brushes
    </a>
  </div>
</section>
`;
